### DISCUSSION TITLE
Does dpgen support the sge job submission system?

### DISCUSSION THREAD

**Discussion by ChengQIAN93** [deepmodeling/dpgen#362] [Q&A] [answered]
Dear Developers
   I want to use dpgen recently. but I found that our job submission system is sge. Does the machine file support sge job submission system? Thank you for your answer.

Cheng Qian

**Comment by AnguseZhang** [comment, marked as answer]
No, for HPC clusters, DP-GEN only supports Slurm, PBS, LSF systems and does not support SGE. But you're welcomed to contribute to DP-GEN if you would.
